awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/Toolbar.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Badge as PFBadge, Button } from '@patternfly/react-core';
+import {
+  BookIcon,
+  CompassIcon,
+  DownloadIcon,
+  RocketIcon,
+  TimesIcon,
+  TrashAltIcon,
+  WrenchIcon,
+} from '@patternfly/react-icons';
+import VerticalSeparator from '@components/VerticalSeparator';
+import styled from 'styled-components';
+
+const Badge = styled(PFBadge)`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-left: 10px;
+`;
+
+function Toolbar({ history, i18n, template }) {
+  const handleVisualizerCancel = () => {
+    history.push(`/templates/workflow_job_template/${template.id}/details`);
+  };
+
+  return (
+    <div>
+      <div
+        style={{
+          borderBottom: '1px solid grey',
+          height: '56px',
+          display: 'flex',
+          alignItems: 'center',
+          padding: '0px 20px',
+        }}
+      >
+        <div style={{ display: 'flex' }}>
+          <b>Workflow Visualizer</b>
```

Comment:
A few missed translations in this file.

Reply:
Ah yea, I kind of threw that component together as a placeholder of sorts - will add a commit for the translations though.

Reply:
See https://github.com/ansible/awx/pull/5379/commits/00970cd43bbe4ab585a8ddd68c77c949b54114e0